packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -24,17 +22,13 @@ @implementation FLTGoogleMapMarkerController
 - (instancetype)initMarkerWithPosition:(CLLocationCoordinate2D)position
                             identifier:(NSString *)identifier
                                mapView:(GMSMapView *)mapView
-                             iconImage:(UIImage *)iconImage
-                              fontPath:(nonnull NSString *)fontPath
-                              markerRadius:(CGFloat)radius {
+                            cozyMarkerBuilder:(nonnull CozyMarkerBuilder *)cozy {
     self = [super init];
     if (self) {
+        _cozy = cozy;
```

Comment:
this file itself seems big in changes but i just removed most code and moved to CozyMarkerBuilder!